invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -0,0 +1,189 @@
+        return [
+            c['person_or_org']['name']
+            for c in obj['metadata'].get('creators', [])
+        ]
+
+    def get_relations(self, obj):
+        """Get relations."""
+        rels = []
+
+        # Fundings
+        fundings = obj['metadata'].get('funding', [])
+        for f in fundings:
+            award = f.get('award')
+            if award:
+                rels.append(
+                    u'{0}:{1}'.format(award['scheme'], award['identifier'])
+                )
+
+            funder = f.get('funder', {})
+            if funder:
+                rels.append(
+                    u'{0}:{1}'.format(funder['scheme'], funder['identifier'])
+                )
+
+        # Alternate identifiers
+        for a in obj['metadata'].get('alternate_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        # Related identifiers
+        for a in obj['metadata'].get('related_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        return rels
+
+    def get_rights(self, obj):
+        """Get rights."""
+        rights = []
+        for right in obj['metadata'].get('rights', []):
+            rights.append(right['title'])
+
+            license_url = right.get('link')
+            if license_url:
+                rights.append(license_url)
+
+        return rights
+
+    def get_dates(self, obj):
+        """Get dates."""
+        dates = [obj["metadata"]["publication_date"]]
+
+        if obj['access']['status'] == 'embargoed':
+            dates.append(obj['access']['embargo']['until'])
+
+        return dates or missing
+
+    def get_descriptions(self, obj):
+        """Get descriptions."""
+        metadata = obj['metadata']
+        descriptions = []
+
+        description = metadata.get("description")
+        if description:
+            descriptions.append(description)
+
+        additional_descriptions = metadata.get("additional_descriptions", [])
+        for add_desc in additional_descriptions:
+            descriptions.append(add_desc['description'])
+
+        return descriptions
+
+    def get_subjects(self, obj):
+        """Get subjects."""
+        metadata = obj['metadata']
+        subjects = []
+        subjects.extend(metadata.get('keywords', []))
+        subjects.extend(
+            (
+                s['subject']
+                for s in metadata.get('subjects', [])
+                if s.get('subject')
+            )
+        )
+        return subjects
+
+    def get_publishers(self, obj):
+        """Get publishers."""
+        return [obj['metadata']['publisher']]
+
+    def get_contributors(self, obj):
+        """Get contributors."""
+        return [
+            c['person_or_org']['name']
+            for c in obj['metadata'].get('contributors', [])
+        ]
+
+    def get_types(self, obj):
+        """Get resource type."""
+        type = obj['metadata']['resource_type']
+        return [type['id']]
```

Comment:
Not sure which is the convention, but with `id` you will be sending sort of an internal value that is `type-subtype` (e.g. [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml#L16))... You might want to send a labelled one? I'm not sure about the spec... see for example the [datacite](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/serializers/datacite/schema.py#L197) serializer

Reply:
The resource type vocabulary should provide a mapping to the eu-repo vocabulary, but I don't think this is yet the case, so perhaps this is best we can do for now:

https://guidelines.openaire.eu/en/latest/literature/field_publicationtype.html

Reply:
should we use a labelled type (openaire maybe?) until the eu-repo vocabulary is added?